ui.tests/test-module/specs/title.spec.js

```diff
@@ -0,0 +1,96 @@
+/*
+ *  Copyright 2022 Adobe Systems Incorporated
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+const commons = require('../libs/commons/commons'),
+    sitesSelectors = require('../libs/commons/sitesSelectors'),
+    sitesConstants = require('../libs/commons/sitesConstants'),
+    guideSelectors = require('../libs/commons/guideSelectors'),
+    afConstants = require('../libs/commons/formsConstants');
+
+describe('Page - Authoring', function () {
+
+    const dropTitleInContainer = function () {
+        const responsiveGridDropZone = "Drag components here",
+            responsiveGridDropZoneSelector = sitesSelectors.overlays.overlay.component + "[data-text='" + responsiveGridDropZone + "']";
+        cy.selectLayer("Edit");
+        cy.insertComponent(responsiveGridDropZoneSelector, "Adaptive Form Title component", afConstants.components.forms.resourceType.title);
```

Comment:
you would need to update here also after changing the jcr:title property to "Adaptive Form Title"